jobs/haproxy/templates/certs.ttar.erb

```diff
@@ -44,6 +44,12 @@ if_p("ha_proxy.crt_list") do |crt_list|
     if list_entry.key?("ssl_ciphers")
       sslbindconf += " ciphers " + list_entry["ssl_ciphers"]
     end
+    if list_entry.key?("ssl_min_version")
```

Comment:
Please provide a test for this in https://github.com/cloudfoundry-incubator/haproxy-boshrelease/blob/master/spec/haproxy/templates/certs.ttar_spec.rb

Reply:
Done!